Packs/AWS-Athena/Integrations/AWS-Athena/AWS-Athena.py

```diff
@@ -1,248 +1,299 @@
+
+    if response.get('ResponseMetadata', {}).get('HTTPStatusCode') == 200:
+        return CommandResults(readable_output=f"Query '{query_execution_id}' has been successfully stopped.")
 
-def get_query_execution_command(args):
-    client = aws_session(
-        region=args.get('region'),
-        roleArn=args.get('roleArn'),
-        roleSessionName=args.get('roleSessionName'),
-        roleSessionDuration=args.get('roleSessionDuration'),
+    else:
+        demisto.debug("Response:\n" + str(response))
+        raise DemistoException(f"Failed to stop query '{query_execution_id}'.")
+
+
+def get_query_execution_command(args: dict, client):
+    query_execution_id: str = args['QueryExecutionId']
+    response = get_query_execution(client=client, query_execution_id=query_execution_id)
+
+    return CommandResults(
+        outputs_prefix=f'AWS.Athena.{QUERY_DATA_OUTPUTS_KEY}',
+        outputs_key_field='QueryExecutionId',
+        outputs=response,
+        raw_response=response,
+        readable_output=tableToMarkdown('AWS Athena Query Execution', response),
     )
-    kwargs = {'QueryExecutionId': args.get('QueryExecutionId')}
-    response = client.get_query_execution(**kwargs)
-    try:
-        raw = json.loads(json.dumps(response, cls=DatetimeEncoder))
-    except ValueError as e:
-        return_error('Could not decode/encode the raw response - {err_msg}'.format(err_msg=e))
-    ec = {'AWS.Athena.Query(val.QueryExecutionId === obj.QueryExecutionId)': raw}
-    human_readable = tableToMarkdown('AWS Athena Query', raw)
-    return_outputs(human_readable, ec)
-
-
-def get_query_results_command(args):
-    client = aws_session(
-        region=args.get('region'),
-        roleArn=args.get('roleArn'),
-        roleSessionName=args.get('roleSessionName'),
-        roleSessionDuration=args.get('roleSessionDuration'),
+
+
+def get_query_results_command(args: dict, client):
+    query_execution_id: str = args['QueryExecutionId']
+    response = get_query_results(client=client, query_execution_id=query_execution_id)
+
+    return CommandResults(
+        outputs_prefix=f'AWS.Athena.{QUERY_RESULTS_OUTPUTS_KEY}',
+        outputs=response,
+        raw_response=response,
+        readable_output=tableToMarkdown('AWS Athena Query Results', response),
     )
-    kwargs = {'QueryExecutionId': args.get('QueryExecutionId')}
-    response = client.get_query_results(**kwargs)
-    ec = {'AWS.Athena.Query(val.QueryExecutionId === obj.QueryExecutionId)': response}
-    human_readable = tableToMarkdown('AWS Athena Query', response)
-    return_outputs(human_readable, ec)
 
 
-"""COMMAND BLOCK"""
-try:
-    LOG('Command being called is {command}'.format(command=demisto.command()))
-    if demisto.command() == 'test-module':
-        client = aws_session()
-        response = client.list_named_queries()
-        if response['ResponseMetadata']['HTTPStatusCode'] == 200:
-            demisto.results('ok')
+@polling_function(
```

Comment:
it's look like `hide_polling_output` is missed in yml
see the documentation of `polling_function`